src/main/java/seedu/address/model/person/Payment.java

```diff
@@ -0,0 +1,56 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * Represents a Student's payment status in the address book.
+ */
+public class Payment {
+
+    public final Boolean hasPaid;
+
+    /**
+     * Constructs an {@code Address}.
+     *
+     * @param hasPaid Boolean for payment status
+     */
+    public Payment(Boolean hasPaid) {
+        requireNonNull(hasPaid);
+        this.hasPaid = hasPaid;
+    }
+
+    /**
+     * Constructs an {@code Address}.
+     * Set to false as default
+     */
+    public Payment() {
+        this.hasPaid = false;
+    }
```

Comment:
hasPaid is final, should it be not final instead? Or should there be a constructor argument?